scripts/dockerfiles/Dockerfile.release

```diff
@@ -20,6 +20,9 @@ COPY ["LICENSE", "NOTICE", "/"]
 
 COPY out/amazon-ecs-pause.tar /images/amazon-ecs-pause.tar
 
+# Copy our cni plugins ecs-eni, ecs-ipam and ecs-bridge
+COPY out/cni-plugins /cni-plugins
```

Comment:
Thanks for catching that Peng. I'd like to retain the path as `"/amazon-ecs-cni-plugins"`. So, it'd be awesome if you could modify this dockerfile to fit that instead of the other way around.